.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,146 @@
+name: "Continuous Integration"
+
+on: [push, pull_request]
+
+jobs:
+
+  phpunit-sqlite:
+    name: "PHPUnit with SQLite"
```

Comment:
The names seem to have versions appended automatically, right? e.g. this one comes out as "PHPUnit with SQLite (5.6)"

Is there any way to control that? For the MariaDB tests, it's not clear which number refers to which thing.

Reply:
Yes, it's automatic by default. It seems that if you include ${{ matrix.X}} values in the name, you can overwrite this,  e.g.

`name: "PHPUnit with MariaDB (${{ matrix.mariadb-version }}). PHP: ${{ matrix.php-version }}, extension: ${{ matrix.extension }}, experimental: ${{ matrix.experimental }}"`

gives "PHPUnit with MariaDB (10.3). PHP: 5.4, extension: pdo_mysql, experimental: false" etc.

(https://github.com/ElliottKasoar/gocdb/actions/runs/990052868)

The names don't need to include every matrix value, or even be unique, e.g.

`name: "PHPUnit with MariaDB (${{ matrix.mariadb-version }})"`

gives "PHPUnit with MariaDB (10.3)" for all of them.

(https://github.com/ElliottKasoar/gocdb/runs/2962602044)

Should I commit something like the former (possibly without experimental?)?

Reply:
>  (possibly without experimental?)

Yes I don't think we need that.

Reply:
Would you include the SQLite version even though it's not explicitly set?

Reply:
How does SQLite get "installed" (for lack of a better term) in the environment the test runs in?

Is it simply there in the `ubuntu-18.04` VM? Does doctrine install it?

Reply:
It seems to come pre-installed:
https://github.com/actions/virtual-environments/blob/ubuntu18/20210628.1/images/linux/Ubuntu1804-README.md

Reply:
Then maybe just a comment in the yml file pointing people to the [documentation](https://github.com/actions/virtual-environments/tree/ubuntu18/20210628.1/images/linux) is sufficient. Unless it can be figured out programmatically, putting any version number for SQLite in the job name is just going to get out of date.

I do agree that something like the below is easier to read than the default.

`name: "PHPUnit with MariaDB: ${{ matrix.mariadb-version }}, PHP: ${{ matrix.php-version }}, extension: ${{ matrix.extension }}."`